daffodil-lib/src/main/scala/org/apache/daffodil/api/DaffodilTunables.scala

```diff
@@ -93,6 +93,12 @@ case class DaffodilTunables(
   //
   val requireFloatingProperty: Boolean = false,
   //
+  // If true, require that the emptyElementPolicy property is specified. If
+  // false, warn about missing property
+  //
+  val requireEmptyElementPolicyProperty: Boolean = false,
```

Comment:
Note name emptyElementPolicy subject to change along with the values of the property.

Reply:
Changed to emptyElementParsePolicy.